README.md

```diff
@@ -7,7 +7,7 @@ Provides useful Halogen Hooks and other utilities commonly used in hooks (e.g. a
 - `useDebouncer` - run an action only when a specified amount of time has passed without receiving any new updats/values (e.g. search the database using user's query after 300 ms of no input).
 - `useThrottle` - run an action with the next value after ignoring new values for a specified amount of time (e.g. handle 1 scroll event every 300 ms).
 - `useEvent` - run a callback whenever a new value is pushed (e.g. allow a library to publish events within it to end-users of the library).
-- `useStateFn` - use the `state /\ modifyState <- useState` behavior that was implemented in `halogen-hook` `v0.3.0` and then reverted in `v0.4.0`.
+- `useStateFn` / `useModifyState` / `usePutState` - allows `state /\ modifyState <- useState` behavior that was implemented in `halogen-hook` `v0.3.0` and then reverted in `v0.4.0`.
 - `useGet` - ensure you aren't getting/using "stale" values (e.g. running finalizers in `Effect`ful code or running asynchronous code).
```

Comment:
Might be worth noting this is specifically for component input, but not for state.

Reply:
Going to let @JordanMartinez tackle adding this `useGet` note. (Either in this PR or another one).

Reply:
Can we reword that to 
```
`useGet` - ensure you aren't getting "stale" `input` values (e.g. referring to the `input` value in an `Effect`'s finalizer or when running asynchronous code).
```

Reply:
Or:

> `useGet` - Produce a `get` function for a value by saving the value in a mutable reference. This function behaves like the `Hooks.get` function for reading the current state. Most commonly used to get a mutable reference to component `input` so you can read the most up-to-date input when in a callback or effect cleanup.

Reply:
Yeah, that's a better wording. Let's use @thomashoneyman's rendering.